### ISSUE TITLE
Example Scene broken

### ISSUE THREAD

**Issue by artfail** [Looking-Glass/JoyconLib#35]
MacOS Catalina 10.15.3
Unity 2019.3.5

The example scene is broken. It is missing scripts and missing textures.
- John

**Comment by BadToxic** [comment]
For me from the state of today on Windows and Unity 2019.3.7 there doesn't seem to be anything missing. Why not tell us what exactly is missing?

**Comment by artfail** [comment]
Screenshots taken today
Mac osx 10.15.4
Unity 2019.3.9
Cube, Cube2, JoyconManager missing scripts.

<img width="1392" alt="Screen Shot 2020-04-13 at 11 00 55 PM" src="https://user-images.githubusercontent.com/10361026/79181798-42c8ff00-7ddb-11ea-9459-1bb8eaca4392.png">
<img width="1392" alt="Screen Shot 2020-04-13 at 11 01 38 PM" src="https://user-images.githubusercontent.com/10361026/79181820-4ceafd80-7ddb-11ea-8409-f9af09bdb67e.png">

**Comment by alxdncn** [comment]
Are you using the Universal Render Pipeline @artfail?

**Comment by artfail** [comment]
Nope. This is what it looks like right out of the box. Just opening the project. No changes.
- John
